native/cocos/renderer/gfx-wgpu/WGPUBuffer.h

```diff
@@ -24,8 +24,9 @@
 ****************************************************************************/
 
 #pragma once
-#include <emscripten/bind.h>
-#include <emscripten/val.h>
+#ifdef CC_WGPU_WASM
+    #include "WGPUDef.h"
+#endif
```

Comment:
Why need this macro in webgpu implementation?

Reply:
in order to support google-dawn-based implementation, instead of wasm by emscripten

Reply:
Got it.